docs/3.0.0-beta.x/plugins/users-permissions.md

```diff
@@ -1,8 +1,47 @@
-# Authentication
+# Users & Permissions
 
-This Authentication API requires the Users & Permissions plugin which comes with Strapi, installed by default.
+This plugin provide a way to protect your API with a full authentication process based on JWT. This plugin comes also with an ACL strategy that allows you to manage the permissions between the groups of users.
 
-## Token usage
+To acccess to the plugin admin pannel to manage the plugin, click on the **Users & Pemissions** link in the left menu.
+
+## Concept
+
+When this plugin is installed, it adds an access layer on your application.
+The plugin use [`jwt`](https://fr.wikipedia.org/wiki/JSON_Web_Token) system to authenticate users.
+
+Each time an API request is made, we are checking if an `Authorization` header is set and strapi verify is you can acces or not to this URL.
+
+To do so, your JWT contain your user ID and we are able to match the group your user is in and at the end to know if the goup allow to acces to the route.
+
+## Manage roles permissions
+
+### Public role
+
+This role is used when you receive a request that don't have `Authorization` header.
+If you allow some permissions in this role, everybody will be able to access to the endpoints you opened.
+It's a good point if you want to let your front-end application access freely to your content.
+
+### Authenticated role
+
+This role is the default one that is apply by default to your **Users**. In this role you will be able to define routes that a user can access.
+
+### Permissions management
+
+By clicking on the **Role** name, you will be able to see all functions available in your application (and these function are related to a specific route)
+
+If you check a fonction name, it makes this route accessible by the current role you are editing.
+On the right sidebar you will be able to see the URL related to this function.
+
+### Update the default role
+
+When you create a user without role or if you use the register route, strapi apply the default role to this new user.
```

Comment:
When you create a user without a role or use the `/register` route, the `authenticated` role is given to the user.

To change the default role, go to the `Advanced settings` tab and update the `Default role for authenticated users` option.